spyder/config/main.py

```diff
@@ -636,4 +636,4 @@
 #    or if you want to *rename* options, then you need to do a MAJOR update in
 #    version, e.g. from 3.0.0 to 4.0.0
 # 3. You don't need to touch this value if you're just adding a new option
-CONF_VERSION = '58.1.0'
+CONF_VERSION = '59.0.0'

```

Comment:
This needs to be bumped to `60.0.0` because I already bumped it to 59 in another PR.